includes/configs/styles.php

```diff
@@ -11,6 +11,12 @@
 defined( 'ABSPATH' ) || exit;
 
 return [
+	'color_picker'      => [
+		'handle' => 'wp-color-picker',
+		'src'    => '',
```

Comment:
Why "src" is empty, or this style is just used for inline styles?

Reply:
Yes, it is just needed to switch on styles